mindmeld/models/taggers/taggers.py

```diff
@@ -181,22 +181,22 @@ def _predict_proba(X):
         del X
         pass
 
+    @property
+    def is_serializable(self):
+        # The default is True since < MM 3.2.0 models are serializable by default
+        return True
+
     @staticmethod
-    def dump(model_path, config):
+    def dump(model_path):
         """
-        Since traditional SKLearn models are easily serializable, we can
-        use JobLib to serialize them. So we alter the context object to make
-        this explicit.
+        Dumps the model to memory. This is a no-op since we do not
```

Comment:
Why are we dumping to memory? Shouldnt we dumping to disk?

Reply:
typo corrected